lib/airports.rb

```diff
@@ -12,6 +12,10 @@ def self.find_by_iata_code(iata_code)
     Airport.
       new(airport_data.each_with_object({}) { |(k, v), h| h[k.to_sym] = v })
   end
+  
+  def self.find_all_by_city(city)
```

Comment:
Can we rename this to `by_city`?  That feels slightly nicer. It'd also be good to have some tests.